src/01-functions.js

```diff
@@ -104,15 +118,21 @@ function averageThree() {}
  * @returns {boolean}
  *
  * HINT: What JavaScript function returns the data type of something?
- * 
+ *
  * EXAMPLE:
  *  compareTypes("left", "right");
  *  //> true
  *
  *  compareTypes("left", 10);
  *  //> false
  */
-function compareTypes() {}
+function compareTypes(a, b) {
+  if (typeof a == typeof b) {
```

Comment:
This will work for this problem, but if you want to give yourself a little bonus I want to challenge you to think of some cases where this would not work and how you would refactor to handle that programmatically!